src/main/java/seedu/storage/LoginStorage.java

```diff
@@ -20,8 +22,20 @@ static void load() {
                 Storage.password = Integer.parseInt(passwordHash);
                 return;
             }
+            createNewUser();
```

Comment:
Not sure, but maybe put this in an else statement to increase readability.

Reply:
makes sense, have rectified, ty